postmortems/README.md

```diff
@@ -0,0 +1,27 @@
+# Postmortem Process
+
+## When to write a postmortem
+
+Project maintainers may decide to write a postmortem when:
+- a project fails to conform to its stability policies
+- users can't cleanly upgrade to a new release
+- a workflow that worked before a release breaks after upgrading
+- any time they want to learn from a failure in existing community processes and improve them
+
+## Goals of a postmortem
+
+The goal of a postmortem is NOT to identify people to blame for a problem.
+We use postmortems to identify opportunities to remove human error from our systems.
+Postmortems should cover:
+- How we will investigate and repair the existing incident?
+- How can we detect similar incidents in the future more quickly?
+- How can we reduce the impact of similar incidents?
+- How can we prevent similar incidents from happening at all? 
+
+## How to create a new postmortem
+
+Copy the [postmortem template](./postmortem-template.md.template) to a new file in this folder
```

Comment:
Having a post mortem template is fine (you could also make this a template in the community drive), but I'm skeptical of how useful it will be to add process around this in the community repo - I'd lean towards letting projects be self-sufficient and not requiring gov-board approval for changes.

e.g. for the v0.43 doc you're working on, I'd be happy with linking to the google doc you already wrote from an issue in pipelines repo with a issue task list.

Reply:
That's a good point. Maybe we could create a google doc template, and a folder for postmortems in the Tekton google drive?

Reply:
Works for me!

Reply:
thanks Billy! I created a folder + template in drive but kept the instructions around, lmk what you think.